app/src/main/java/ca/ualbert/cs/tasko/NotificationArtifacts/NotificationType.java

```diff
@@ -15,29 +15,10 @@
 
 package ca.ualbert.cs.tasko.NotificationArtifacts;
 
-import ca.ualbert.cs.tasko.User;
-import io.searchbox.annotations.JestId;
-
 /**
- * Created by spack on 2018-03-10.
+ * Created by spack on 2018-03-27.
  */
 
-public class SimpleNotification extends Notification{
-
-
-    public SimpleNotification(String message, User recipient){
-        super(message, recipient);
-    }
-
-    @JestId
-    private String id;
-
-    public void setId(String id) {
-        this.id = id;
-    }
-
-    public String getId() {
-        return id;
-    }
-
+public enum NotificationType {
+    TaskRequesterRecievedBidOnTask, TaskProviderBidAccepted, TaskProviderBidDeclined, Rating, TaskDeleted
```

Comment:
Common Java Coding Convention says ENUM_SHOULD_BE_UPPER_SEPERATED_BY_UNDERSCORE. lol
Also we need 2 rating notifications do we not? One for rating a requester and one for rating a provider?